pkg/products/rhssouser/reconciler.go

```diff
@@ -225,9 +245,334 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, installation *inte
 	}
 	r.logger.Infof("The operation result for keycloakrealm %s was %s", kcr.Name, or)
 
+	// We want to update the master realm by adding an openshift-v4 idp. We can not add the idp until we know the host
+	// url - kc.Status.InternalURL
+	if kc.Status.InternalURL != "" {
```

Comment:
would it simplify things to have ``` if kc.Status.InternalURL == ""{ return } ``` I think we could get rid of the else at the end then also

Reply:
ok, will do